collectors/apps.plugin/apps_plugin.c

```diff
@@ -3006,6 +3008,12 @@ static inline void aggregate_pid_fds_on_targets(struct pid_stat *p) {
         if(likely(fd <= 0 || fd >= all_files_size))
             continue;
 
+
+        currentfds = p->fds_size;
```

Comment:
I wrote that you can't use `fds_size`. You should count for open files here and update `currentmaxfds` after the loop has finished.